server/bin/pbench-index.py

```diff
@@ -36,6 +37,28 @@
 from pbench.server.utils import rename_tb_link, quarantine
 
 
+class CustomError1(Exception):
+    pass
+
+
+class CustomError2(Exception):
+    pass
+
+
+def handler1(signum, frame):
```

Comment:
I would name the handlers and the exceptions they raise less generically: `sigint_handler` would raise `SigIntException` (it's not an error so why add `Error` to the name?). and `sigterm_handler` with `SigTermException`.

Reply:
Thanks, Nick; I actually said the same thing, but I just discovered this morning that I got distracted and never submitted my comments yesterday!

Reply:
Welcome to the club! Too many tabs open in too many browser windows...